client/app/scripts/superdesk-desks/views/aggregate-settings.html

```diff
@@ -36,8 +36,11 @@ <h3 translate>Monitoring settings</h3>
 
     <div sd-wizard-step data-title="{{ :: 'Saved Searches' | translate }}" data-code="searches">
         <div class="content">
-            <div class="legend" translate>Select saved searches for view</div>
-            <div class="desk" ng-repeat="search in searches | filter: {'user': user} | orderBy: 'name'">
+            <div class="legend" translate>Select saved searches for view.  
+                <span sd-switch class="pull-right" ng-model="allSavedSearches" 
+                 ng-click="initSavedSearches(allSavedSearches)" title="Show all saved searches"></span>
+            </div>
+            <div class="desk" ng-repeat="search in currentSavedSearches | orderBy: 'name'">
```

Comment:
Do `search` objects have any unique property to track them by it? Because the `currentSavedSearches` list tends to change. Just a thought....